usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -81,7 +83,34 @@ To reduce unnecessary data transfer, most endpoints return a default set of info
 | [/api/v1/submissions/](/api/v1/submissions/) | GET, POST | Returns a list of submissions |
 | [/api/v1/filter/](/api/v1/filter/) | POST | Returns a hash of the json body |
```

Comment:
The following endpoints are getting 404 errors:
`/api/v1/awards/autocomplete/`
`http://localhost:8000/api/v1/hash/`
`http://localhost:8000/api/v1/filter/`

Reply:
removed from endpoints.md